appium/options/android/common/mjpeg/mjpeg_screenshot_url_option.py

```diff
@@ -19,7 +19,7 @@
 
 from appium.options.common.supports_capabilities import SupportsCapabilities
```

Comment:
should we also update this option and below ones?

Reply:
Yes I have changed it to use descriptor.